server/game/gamesteps/uiprompt.js

```diff
@@ -14,14 +14,25 @@ class UiPrompt extends BaseStep {
 
     complete() {
         this.completed = true;
+        if(this.getPlayer()) {
+            // if(this.getPlayer().stopClock) {
```

Comment:
Current build is failing due to linting errors. Remove the commented out ifs and re-indent.

Reply:
thanks. fixed